frontend-react/src/hooks/UseValueSets.test.ts

```diff
@@ -11,18 +11,35 @@ import {
 } from "../network/api/LookupTableApi";
 import { QueryWrapper } from "../utils/CustomRenderUtils";
 
-import { useValueSetsTable } from "./UseValueSets";
+import {
+    useValueSetsTable,
+    useValueSetUpdate,
+    useValueSetActivation,
+} from "./UseValueSets";
 
 describe("useValueSetsTable", () => {
+    const renderWithQueryWrapper = (
+        tableName: LookupTables,
+        version?: number
+    ) =>
+        renderHook(() => useValueSetsTable<ValueSet>(tableName, version), {
```

Comment:
The one thing I'd say we should do here to make it reusable is make the parameter take `(props?: any) => any` (i.e. a hook) as the parameter in `renderWithQueryWrapper`. Non-blocking for this PR, but wanna make sure we're pulling these types of tools out in a final review of the `shadow-master` ☠️

Reply:
Here's the docs with types for `renderHook()`: https://react-hooks-testing-library.com/reference/api#renderhook